Fabric.Authorization.AccessControl/src/app/services/interceptors/fabric-http-request-interceptor.service.ts

```diff
@@ -38,6 +38,10 @@ export class FabricHttpRequestInterceptorService implements HttpInterceptor {
       );
 
       return tokenObservable.pipe(mergeMap(accessToken => {
+        // do not send request if user token was not found (not authenticated), and was required
+        if (!accessToken) {
+          return empty();
```

Comment:
Prevent trying to send a request to discovery without an auth token and showing a windows auth prompt for a moment.

Caused because the auth.service is trying to populate service.service's service list by calling discovery, which fires off this interceptor, but auth.service has yet to be fully initialized so it can't quite get a user in time to add an access token.

Reply:
Should we also just add a console log here, so we know that we blocked a request?

Reply:
Sure